Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:12
+WORKDIR /app
+COPY . /app/
+RUN npm install
+EXPOSE 3000
+RUN echo $(date +%s) > RELEASE_ID
+RUN APOS_RELEASE_ID=$(cat RELEASE_ID); npm run build
+ARG APOS_RELEASE_ID=$(cat RELEASE_ID)
```

Comment:
ARG "is not persisted in the final image." Rather than trying to avoid RELEASE_ID going into the container, we should embrace it going into the container, so you can do APOS_RELEASE_ID=$(cat RELEASE_ID) in the CMD line too.

Also you don't need a `;` after setting a one-off environment variable for a command.